test/Microsoft.ML.Tests/Transformers/CustomTransformerTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Data.IO;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Tools;
+using System.ComponentModel.Composition;
+using System.ComponentModel.Composition.Hosting;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Tests.Transformers
+{
+    public sealed class CustomTransformerTests : TestDataPipeBase
```

Comment:
(nit) I would align the test class name with the name of the class it is testing.

`CustomMappingEstimator` and `CustomMappingTransformer`.  I would call this class `CustomMappingTransformerTests`.

Reply:
I used `CustomMappingTests` since we are not always using `Estimator` or `Transformer` suffix in test names, and I don't think we should.

---
In reply to: [229417149](https://github.com/dotnet/machinelearning/pull/1406#discussion_r229417149) [](ancestors = 229417149)